README.md

```diff
@@ -1,4 +1,19 @@
-# fcli
-Foundational Components CLI
-
+ - Foundational Components CLI
 Helps spread the AwesomeSauce of the Foundational Components team a bit further.
+
+
+## Getting Started
+We recommend using Python 3.
+
+cd fcli
+python3.6 setup.py install
```

Comment:
Can you put triple backticks around this to monospace the shell commands here?  As it stands, the two lines here are merged into one line when viewing.